charts/opensearch/CHANGELOG.md

```diff
@@ -13,6 +13,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 ### Security
 ---
```

Comment:
Please add links to Unreleased section and release links at the bottom of CHANGELOG

Reply:
Done.